CMakeLists.txt

```diff
@@ -1,8 +1,6 @@
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 3.13)
 
-if("${CMAKE_VERSION}" VERSION_GREATER 3.0.0)
-    cmake_policy(SET CMP0042 NEW)
-endif()
+cmake_policy(SET CMP0042 NEW)
```

Comment:
I don't think this is needed anymore now that we specify the minimum version to 3.13